config/routes.rb

```diff
@@ -266,7 +266,9 @@ def external_redirect(path_prefix, target)
           resources :translations, controller: "edition_translations", except: %i[index show] do
             get :confirm_destroy, on: :member
           end
-          resources :editorial_remarks, only: %i[new create], shallow: true
+          resources :editorial_remarks, shallow: true do
```

Comment:
Should this be 

```
resources :editorial_remarks, only: %i[new create destroy], shallow: true
```

So it doesn't create the index, show etc. endpoints?

Reply:
Yes, thanks!